docs/bash-completion

```diff
@@ -1,1101 +0,0 @@
-
-# Copyright 2016 The Kubernetes Authors.
```

Comment:
The command `minikube completion` comes up in the --help reference for minikube and generates this file.